src/pages/Controllers/Controllers.js

```diff
@@ -185,7 +185,8 @@ function RegisterAController({ onClose }) {
     <SlidePanel onClose={onClose}>
       <h5>Register a Controller</h5>
       <p className="p-form-help-text">
-        Controller information can be retrieved using the `juju show-controller`
+        Controller information can be retrieved using the{" "}
```

Comment:
This pushes this over multiple lines and I was trying to keep it on one. Can you up the max-width of slide panel to 600px to accommodate?

Reply:
560px is enough